typescript/infra/scripts/get-key-addresses.ts

```diff
@@ -0,0 +1,28 @@
+import { getAllKeys } from '../src/agents/key-utils';
+
+import { getAgentConfig, getEnvironment } from './utils';
+
+async function main() {
+  const environment = await getEnvironment();
+  const agentConfig = await getAgentConfig(environment);
+
+  const keys = getAllKeys(agentConfig);
+
+  const keyInfos = await Promise.all(
+    keys.map(async (key) => {
+      let address = '';
+      try {
+        await key.fetch();
+        address = key.address;
+      } catch (e) {}
+      return {
+        identifier: key.identifier,
+        address,
+      };
+    }),
+  );
+
+  console.log('Keys:', JSON.stringify(keyInfos, null, 2));
```

Comment:
It might be nice for either this to be "pure" JSON output so that it can be piped into JQ or for it to print it out "prettty" line by line